tz.cpp

```diff
@@ -126,7 +126,7 @@ CONSTDATA auto max_day = date::dec/31;
 
 namespace detail
 {
-class undocumented {explicit undocumented() = default;};
+class undocumented {public: explicit undocumented() = default;};
```

Comment:
Wow, I don't know how that compiled.  Instead of putting public in there, lets change it to a `struct`.